File: LAPORAN PRATIKUM/MODUL7/CODEBLOKS DAN PYTHON/SOAL 3/PRAK603-2210817210014-MuhammadQoharyRawidafany.c
#include <stdio.h>
int main ()
{
    int bil1[20],bil2[20],bar,bar2,total;
    scanf("%d %d",&bar,&bar2);
    if (bar == bar2){
    for (int a=0;a<bar;a++){
        scanf("%d",&bil1[a]);
    }
    for (int a=0;a<bar2;a++){
        scanf("%d",&bil2[a]);
        }
    for (int a=0;a<bar2;a++){
            total = bil1[a] * bil2[a];
        printf("%d ",total);
    }
    }
    else {
        printf("Jumlah tidak Sama");
    }
}

File: LAPORAN PRATIKUM/MODUL7/CODEBLOKS DAN PYTHON/SOAL 3/PRAK603-2210817210014-MuhammadQoharyRawidafany.py
n1,n2 = map(int,input().split())
if n1 == n2 :
    bil1= list(map(int, input().split()))
    bil2= list(map(int, input().split()))
    for a in range (n2):
        hasil = bil1[a] * bil2[a]
        print(hasil,end =' ')
else : print("Jumlah tidak sama")